### DISCUSSION TITLE
Replicas on different workernodes

### DISCUSSION THREAD

**Discussion by ilabrovic** [rabbitmq/cluster-operator#1645] [Q&A] [answered]
We are using the RabbitMq cluster operator and are deploying a rabbitmq cluster with the rabbitmqcluster resource.
From a lot of our deployments, we see that 2 of 3 replicas are running on the same worker, that is not desirable.
Are there any options/flags in de rabbitmqcluster resource i can set, so that each replica is running on a different workernode?

**Comment by mkuratczyk** [comment, marked as answer]
All the usual Kubernetes features are available, for example
https://www.rabbitmq.com/kubernetes/operator/using-operator#affinity
https://github.com/rabbitmq/cluster-operator/tree/main/docs/examples/pod-anti-affinity

**Reply by ilabrovic** [reply]
Perfect! Tested and works just what we were looking for!
